src/chapters/enums/enums-in-models.rst

```diff
@@ -0,0 +1,191 @@
+   If you ever want to verify what code the video ends with, check out the `enums <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/enums>`__ branch in ``CodingEventsDemo``.
+
+Enum Types in Models - Text
+---------------------------
+
+Create an Enum Class
+^^^^^^^^^^^^^^^^^^^^
+
+In your ``Models`` directory within ``CodingEvents``, create a new class called ``EventType``. 
+Replace the ``class`` keyword with ``enum`` and delete the default constructor.
+
+Because enum values are constants, we use naming conventions and write them in all caps.
+Each value is demarcated with a comma and the list is completed with a semicolon.
+
+``EventType``:
+
+.. sourcecode:: csharp
+   :lineno-start: 4
+
+   public enum EventType
+   {
+      CONFERENCE,
+      MEETUP,
+      WORKSHOP,
+      SOCIAL
+   }
+
+Add an Enum Property to a Model Class
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Other objects can have enum type properties. To add an ``EventType`` property to our model ``Event``,
+we create a ``Type`` property in ``Event`` amongst the other properties declared:
+
+.. sourcecode:: csharp
+   :lineno-start: 12
+
+   // other Event property declarations
+
+   public EventType Type { get; set; }
+
+   // Event methods
+
+Once we have added an ``EventType`` property to our model, we need to add an ``EventType`` property to our ViewModel.
+
+Add an Enum Property to a ViewModel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first thing we need to do is add an ``EventType`` property to ``AddEventViewModel``.
+
+.. sourcecode:: csharp
+   :lineno-start: 20
+
+   // other AddEventViewModel property declarations
+
+   public EventType Type { get; set; }
+
+Now that we have a property to hold the event type that the user selects, we need to create a list of all of the possible event types.
+We will use this list to populate the ``select`` element in a later step.
+
+.. sourcecode:: csharp
+   :lineno-start: 26
+
+   public List<SelectListItem> EventTypes { get; set; }
+
+   public AddEventViewModel()
+   {
+      EventTypes = new List<SelectListItem>();
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.CONFERENCE.ToString(),
+            Text = EventType.CONFERENCE.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.MEETUP.ToString(),
+            Text = EventType.MEETUP.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.SOCIAL.ToString(),
+            Text = EventType.SOCIAL.ToString()
+      });
+
+      EventTypes.Add(new SelectListItem
+      {
+            Value = EventType.WORKSHOP.ToString(),
+            Text = EventType.WORKSHOP.ToString()
+      });
+   }
+
+In the ``AddEventViewModel()`` constructor, we add each of the constants to the ``EventTypes`` list.
```

Comment:
This is another great example of why we use ViewModels, and is worth calling out! The view needs a list of all of the event types, but we would never want to have such a list on `Event`.

Reply:
Mentioned it!